Apps/phcybereason/cybereason_consts.py

```diff
@@ -0,0 +1 @@
+# Define your constants here
```

Comment:
Delete this file since not being used. Alternatively you can move your JSON templates here